test/integration/test_cli_commands.py

```diff
@@ -874,3 +874,20 @@ def test_logging_with_logformat_logstash_flag(self):
                     obj={"filters":[]})
         output = sorted(result.output.splitlines(), reverse=True)[:4]
         self.assertEqual(expected, output)
+
+class TestCLIRepositoryCreate(CuratorTestCase):
+    def test_create_fs_repository(self):
+        test = clicktest.CliRunner()
+        result = test.invoke(
+                    curator.repomgrcli,
+                    [
+                        '--logfile', os.devnull,
+                        '--host', host,
+                        '--port', str(port),
+                        'create',
+                        'fs',
+                        '--repository', self.args['repository'],
+                        '--location', '/tmp'
```

Comment:
Good point,

One issue is that we need to know the tempdir of the _target_ OS, not our own (target: where ES is running).